projects/packages/forms/tests/php/contact-form/Contact_Form_Endpoint_Test.php

```diff
@@ -139,12 +139,19 @@ public function test_get_integrations_v1_returns_200() {
 		$this->assertArrayHasKey( 'akismet', $data );
 		$this->assertArrayHasKey( 'creative-mail-by-constant-contact', $data );
 		$this->assertArrayHasKey( 'zero-bs-crm', $data );
+		$this->assertArrayHasKey( 'google-drive', $data );
 
 		// Verify structure of one integration
 		$this->assertArrayHasKey( 'type', $data['akismet'] );
 		$this->assertArrayHasKey( 'isInstalled', $data['akismet'] );
 		$this->assertArrayHasKey( 'isActive', $data['akismet'] );
 		$this->assertArrayHasKey( 'isConnected', $data['akismet'] );
+
+		// Verify structure of google-drive
+		$this->assertArrayHasKey( 'type', $data['google-drive'] );
+		$this->assertArrayHasKey( 'isInstalled', $data['google-drive'] );
+		$this->assertArrayHasKey( 'isActive', $data['google-drive'] );
+		$this->assertArrayHasKey( 'isConnected', $data['google-drive'] );
```

Comment:
should we add the other fields here. 
Such as 

slug, settingsUrl, pluginFile, version, details?

Reply:
Yes, makes sense. Will work on adding that shortly.

Reply:
I pushed an update to test all properties for both all and single integration endpoints.